src/plugins/guided_onboarding/public/components/guide_panel_step.styles.ts

```diff
@@ -17,17 +17,23 @@ export const getGuidePanelStepStyles = (euiTheme: EuiThemeComputed, stepStatus:
     height: 24px;
     border-radius: 50%;
     border: 2px solid
-      ${stepStatus === 'inactive' ? euiTheme.colors.lightShade : euiTheme.colors.success};
+      ${stepStatus === 'inactive'
+        ? euiTheme.colors.borderBasePlain
+        : euiTheme.colors.accentSecondary};
     font-weight: ${euiTheme.font.weight.medium};
     text-align: center;
     line-height: 1.4;
-    color: ${stepStatus === 'inactive' ? euiTheme.colors.subduedText : euiTheme.colors.text};
+    color: ${stepStatus === 'inactive'
+      ? euiTheme.colors.textSubdued
+      : euiTheme.colors.textParagraph};
   `,
   stepTitle: css`
     font-weight: ${euiTheme.font.weight.semiBold};
-    color: ${stepStatus === 'inactive' ? euiTheme.colors.subduedText : euiTheme.colors.text};
+    color: ${stepStatus === 'inactive'
+      ? euiTheme.colors.textSubdued
+      : euiTheme.colors.textParagraph};
     .euiAccordion-isOpen & {
-      color: ${euiTheme.colors.title};
+      color: ${euiTheme.colors.textHeading};
```

Comment:
How to access this UI:

https://github.com/user-attachments/assets/a9ce34f9-1903-4a86-a947-2b74e8d5427b